src/mmw/js/src/analyze/models.js

```diff
@@ -54,6 +54,33 @@ var AnalyzeTaskModel = coreModels.TaskModel.extend({
     }
 });
 
+var AnalyzeTaskCollection = Backbone.Collection.extend({
+    model: AnalyzeTaskModel
+});
+
+AnalyzeTaskCollection.getSingleton = function(App, aoi) {
+    if (!App.analyzeCollection) {
```

Comment:
Hmm you're right. Good catch! Although previously we were just checking if `analyzeModel` exists, so I just assumed it would work, but apparently I must have missed something.